zerver/openapi/zulip.yaml

```diff
@@ -3255,6 +3301,7 @@ paths:
     # This entry is a hack; it exists to give us a place to put the text
     # documenting the arguments for call_on_each_event and friends.
     post:
+      operationId: real_time
```

Comment:
Should this have an `operationId`?

Reply:
(It seems possible we want this to not appear in something made by `openapi-generator`, which I think might be what happens if we do that?)

Reply:
I removed its tag and operationId and it still has an API entry. I searched the web but couldn't find a method to stop the generator from using an endpoint.